irony-server.rst

```diff
@@ -0,0 +1,98 @@
+============
+irony-server
+============
+
+----------------------------------------
+The back-end server for Emacs irony-mode
+----------------------------------------
+
+:Author: Nicholas D Steeves <nsteeves@gmail.com>
+:Date: 2017-10-01
+:Version: 1.1.0
+:Manual section: 1
+
+
+Synopsis
+========
+
+irony-server [*options*] [*command*] [<*args*>]
+
+Description
+===========
+
+``irony-server`` provides the libclang interface to irony-mode. It uses a
+simple protocol based on S-expression. This server is written in C++ and
+requires the following packages to be installed on your system: CMake >=
+2.8.3, and libclang
+
+Options
+=======
+
+-v, --version
+
+ Show version and exit.
+
+-h, --help
+
+ Show more detailed command help.
+
+-i, --interactive
```

Comment:
Run an interactive command loop.

After giving it some thought, the name is probably not good, but that's how it is.

Reply:
Hmm.  I see how it's interactive like Python's interactive mode, but it exits on error instead of providing a traceback.  That sound probably be documented.  Does exclusively support commands from the irony-server list, or does it also code snippets?

Reply:
Yeah, the interface is quite sensible...but the real consumer is irony.el so I don't care much about making something more useful.

> Does [it] exclusively support commands from the irony-server list, or does it also [support] code snippets?

It supports only the commands.

Reply:
Please squash my fixup when you merge.  I think it communicates your intent clearly and supports what I believe is your reasoning behind "interactive" :-)